### ISSUE TITLE
exodus apps caterogry 

### ISSUE THREAD

**Issue by XXDRD** [Psyhackological/AAA#37]
Where should [this](https://github.com/Exodus-Privacy/exodus-android-app) go?

**Comment by Psyhackological** [comment]
Well currently nowhere,  because it does not have a dark theme. But maybe I'm blind and I cannot find it?

**Comment by Psyhackological** [comment]
So that means you cannot found it neither?

**Comment by RaianIE** [comment]
I can confirm that you are not blind and it actually doesn't have any dark theme. Good app and but doesn't follow your rule.

**Comment by Psyhackological** [comment]
Yeah, unfortunately. I have this app on my smartphone and forced Android to make it dark themeish.

**Comment by Jean-BaptisteC** [comment]
The new 3.0 version has a dark theme only synced with the system :)

**Comment by Psyhackological** [comment]
@Jean-BaptisteC great! I will add it soon. :)
